training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -76,7 +76,7 @@
     "\n",
```

Comment:
<p>Do we really need to have this section? We don't use any feature of Experiment in the notebook, only creating an Experiment trial that's basically unused.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Agree, will remove it